src/components/Pill/Pill.tsx

```diff
@@ -24,6 +24,12 @@ export const useStyles = makeStyles(
     primaryColor: {
       background: theme.palette.primary.main,
     },
+    negative: {
```

Comment:
Nit. Seeing this for the first time, my thought was "negative" and "positive" would switch the brightness/values of the background and text. Could just be me; I was already in this headspace from another badge PR.

It's supposed to mean "positive" like "pass" or "good" and "negative" as "fail" or "bad", right? I think the names are fine, but am commenting in case enough others feel the same that a rename is warranted.

Reply:
You're correct, `positive=good` and `negative=bad`. I preferred this over `success` and `error` because if something is red, it doesn't necessarily mean it's some kind of error.